vector_icons/components/omnibox/browser/vector_icons/pedal.icon

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
nit: maybe our license sentence should be used?

Reply:
Good catch. I copied the file from `product.icon` in the same directory and it has that copyright note. Updated in both. The rest of the files the directory have a different copyright (Mozilla), so I'll leave those as they are.